### ISSUE TITLE
visual styling

### ISSUE THREAD

**Issue by joeblew99** [GoVios/react-components#24]
I am using react-toolbox at the moment, and am trying to work out if this repo is also designed for Material Design.

For example the Chip is a simple example.
https://github.com/GoVios/react-components/tree/master/src/chip

versus

https://github.com/react-toolbox/react-toolbox/tree/dev/components/chip

If you could let me know it would really help me in my project.

thanks in advance.

**Comment by Inkdpixels** [comment]
Hey @joeblew99 Could you maybe rephrase the question? It's a bit hard to guess what exactly you want to know.
